server/src/router.ts

```diff
@@ -202,10 +198,6 @@ export function createRouter(db: IKeyValueStore): Router {
         .route("/journey")
         .get(async (req, res) => {
             const journey = await store.getJourney(res.locals.hexalotId)
-            if (journey === undefined) {
-                res.sendStatus(404)
-                return
-            }
             res.json(journey)
```

Comment:
You're just sending back `undefined` instead? Why?

Reply:
I don't consider it to be an error if they're absent.

Reply:
here it would be an error if the hexalot didn't exist, i suppose